bin/ramalama

```diff
@@ -1,18 +1,10 @@
-#!/usr/bin/env python3
+#!/bin/bash
```

Comment:
did you plan on overwriting bin/ramalama?  Wny not create new file bin/ramalama-demo?

Reply:
Or drop this  file in docs/demo subdir.

Reply:
I will move, although I'm never gonna check in the PR to be honest, it's hacky as hell